packages/api/src/local-server.ts

```diff
@@ -1,39 +1,44 @@
 import * as dotenv from 'dotenv';
-dotenv.config();
 import express from 'express';
-import { APIGatewayProxyEvent } from 'aws-lambda';
 import { getData, listAirnodeAddresses, batchUpsertData, upsertData } from './handlers';
 
+dotenv.config();
 const { PORT } = process.env;
 
-const port = PORT || 8090;
+const port = PORT;
```

Comment:
I think port should have a default value if it's not set as an env variable

```suggestion
const port = process.env.PORT || 8090;
```

Reply:
I put the default value inside the `.env.example` and the README tells you to copy it. Or am I missing something?

Reply:
I guess I just typically leave it out and just use whatever the default is set to. Fine with leaving it as required too